src/components/enter-email/tests/enter-email-integration.test.ts

```diff
@@ -247,4 +281,38 @@ describe("Integration::enter email", () => {
       .expect("Location", PATH_NAMES.ENTER_PASSWORD)
       .expect(302, done);
   });
+
+  it("should redirect to /signed-out with login_required error when user fails re-auth", async () => {
+    process.env.SUPPORT_REAUTHENTICATION = "1";
+    stubConfig = {
+      statusCode: HTTP_STATUS_CODES.BAD_REQUEST,
+      shouldStub: true,
+    };
+
+    nock(baseApi)
```

Comment:
e.g. here we've got `nock` (which I think is the way to go), but also an unused `stubConfig` above